LinhLe2307/Flash_card_app_React_TypeScript
# Sup 8 add tags to flashcard

### In this PR
- The tags are added in the UI

### How to test
- Go to [New Card](http://localhost:5173/card/new), add tags in new card
- Save and go back to [Card list](http://localhost:5173/cards-user/6639e18cc61f8fcd1dc794d1) and see tags
